cli/cmd/initialize/wizard/wizard.go

```diff
@@ -26,42 +34,43 @@ func RunInitAppWizard(projectFolderPath string, a *tool.Tool) (bool, error) {
 		return false, nil
 	}
 
-	answers := fromApp(a)
-	if err := survey.Ask(questions, answers); err != nil {
+	answers := answersFromManifest(m)
+	if err := survey.Ask(questions, &answers); err != nil {
 		return false, err
 	}
 
-	answers.appendAnswersToApp(a)
+	answers.updateManifest(m)
 
 	return true, nil
 }
 
-func getQuestions(a tool.Tool) []*survey.Question {
-	q := []*survey.Question{}
+func getQuestions(m *manifest.Manifest) []*survey.Question {
```

Comment:
We should probably consider making this more like GO with using `New` instead of `get`. this does not  have to be part of this MR 😄

Reply:
I am not sure it is entirely the same as other cases where we might use `new` as the name. I think it might be even more go conventional to simply called `questions`. Especially since it is a private function anyway.